src/myspace/PersonalDetails.jsx

```diff
@@ -0,0 +1,234 @@
+import React, { useState, useEffect } from "react";
+import Cookies from "js-cookie";
+import { BASE_API } from "../config";
+import "./MySpace.css";
+
+
+export default function PersonalDetails() {
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [personalDetails, setPersonalDetails] = useState({});
+  
+  const requestPersonalDetails = {
+    method: "GET",
+    headers: {
+      "Authorization": `Bearer ${Cookies.get("access_token")}`,
```

Comment:
@mtreacy002 sorry I did not realize the token comes from the cookie. We are creating the AuthContext as a wrapper for any auth related details. If we start getting access_token from cookie directly, every other component that we make will have this same logic. It is the responsibility of the context, to abstract these details out. 

Can you send the access_token also from the provider, and use it here?

Reply:
done. It's now added to AuthContext and used in request header from that. 👍